apollo-router/src/uplink/snapshots/apollo_router__uplink__license_enforcement__test__restricted_features_via_config.snap

```diff
@@ -18,12 +18,6 @@ Configuration yaml:
 * APQ caching
   .apq.router.cache.redis
 
-* Subgraph caching
-  .traffic_shaping.experimental_cache
-
-* Subgraph entity caching
-  .traffic_shaping..experimental_entity_caching
-
```

Comment:
Why don't we have the new one ?

Reply:
no idea. @BrynCooke would you know?

Reply:
The config files under test maybe hasn't been updated? 

https://github.com/apollographql/router/blob/dev/apollo-router/src/uplink/testdata

Reply:
fixed in a4141051